content/en/blog/falcosidekick-reponse-engine-part-6-cloud-run.md

```diff
@@ -0,0 +1,189 @@
+---
+title: "Kubernetes Response Engine, Part 6: Falcosidekick + Cloud Run"
+date: 2021-06-16
+author: Batuhan Apaydın
+slug: falcosidekick-reponse-engine-part-6-cloud-run
+---
+
+> _This blog post is part of a series of articles about how to create a `Kubernetes` response engine with `Falco`, `Falcosidekick` and a `FaaS`._
+>
+> See other posts:
+>
+> * [Kubernetes Response Engine, Part 1 : Falcosidekick + Kubeless]({{< ref "/blog/falcosidekick-reponse-engine-part-1-kubeless" >}})
+> * [Kubernetes Response Engine, Part 2 : Falcosidekick + OpenFaas]({{< ref "/blog/falcosidekick-reponse-engine-part-2-openfass" >}})
+> * [Kubernetes Response Engine, Part 3 : Falcosidekick + Knative]({{< ref "/blog/falcosidekick-reponse-engine-part-3-knative" >}})
+> * [Kubernetes Response Engine, Part 4 : Falcosidekick + Tekton]({{< ref "/blog/falcosidekick-reponse-engine-part-4-tekton" >}})
+> * [Kubernetes Response Engine, Part 5 : Falcosidekick + Argo]({{< ref "/blog/falcosidekick-reponse-engine-part-5-argo" >}})
+
+---
+
+Recently, we added two new output type support to the Falcosidekick, and these are [Cloud Functions](https://github.com/falcosecurity/falcosidekick/pull/241), and [Cloud Run](https://github.com/falcosecurity/falcosidekick/pull/243). In today's blog post, we'll talk about how we can set up `Kubernetes Response Engine` running on `GKE (Google Kubernetes Engine)` by using `Cloud Run`.
+
+Let's start by explaining a little bit about what `Cloud Run` is first. `Cloud Run` is a managed compute platform that enables you to run containers that are invocable via requests or events. `Cloud Run` is serverless: it abstracts away all infrastructure management, so you can focus on what matters most — building great applications.
+
+If you want to learn more about them, you can follow this [link](https://cloud.google.com/run/docs).
+
+Today's reference architecture will look like the following picture:
+
+![cloud_run_reference_arch](/docs/images/cloud_run_reference_arch.png)
+
+This demo might be useful for Google Cloud users who might already be using GKE with `Falco` to protect container runtime against malicious behaviors, and wants to take any action for them with `Cloud Run`.
+
+// TOC
+
+## Prerequisites
+
+- gcloud 342.0.0
+- ko 0.8.3
+
+## Tutorial
+
+### Provision GKE (Google Kubernetes Engine) Cluster
+
+We have to create a GKE, so let's start by creating one.
+
+```shell
+$ GKE_CLUSTER_NAME=cloud-run-demo
+$ gcloud container clusters create $GKE_CLUSTER_NAME
+```
+
+### Deploy Cloud Run Function
+
+Once it's finished, we are ready to deploy our Cloud Run function. But before doing that, we have to explain that what the `Cloud Run` function will do. As you can see in the reference architecture image above, this function will delete the pwned Pod. In order to do that, this function should have the appropriate permissions.
```

Comment:
Once GKE is set up, we are ready to deploy our Cloud Run function. But before doing that, let us see what the `Cloud Run` function will do. As you can see in the reference architecture, this function will delete the pwned Pod. In order to do that, this function should have the appropriate permissions.

Reply:
thank you so much, done.